elfpy/agents/agent.py

```diff
@@ -254,13 +250,25 @@ def update_wallet(self, wallet_deltas: wallet.Wallet, market: Market) -> None:
                 )
                 self.wallet[key].amount += value_or_dict.amount
             # handle updating a dict, which have mint_time attached
+            elif key == "borrows":
+                if value_or_dict:  # could be empty
+                    self._update_borrows(value_or_dict)
             elif key == "longs":
                 self._update_longs(value_or_dict.items())
             elif key == "shorts":
                 self._update_shorts(value_or_dict.items())
             else:
                 raise ValueError(f"wallet_key={key} is not allowed.")
 
+    def _update_borrows(self, borrow_summary: wallet.Borrow) -> None:
+        if borrow_summary.start_time in self.wallet.borrows:  #  entry already exists for this mint_time, so add to it
+            self.wallet.borrows[borrow_summary.start_time].borrow_amount += borrow_summary.borrow_amount
+        else:
+            self.wallet.borrows.update({borrow_summary.start_time: borrow_summary})
+        if self.wallet.borrows[borrow_summary.start_time].borrow_amount == 0:
+            # Remove the empty borrow entry from the wallet.
```

Comment:
i think this is an unnecessary comment, which i don't say often :P

Reply:
if there is a problem that removing the empty one solves, that would be a better comment.